stock_measuring_device_zippcube/scripts/measurement.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+curl --header "Content-Type: application/json" \
```

Comment:
this is only for sample I guess?

Reply:
yes, useful for interactive tests, when you don't have the real thing at hand.